docs/connectors/mysql.asciidoc

```diff
@@ -1185,6 +1185,22 @@ Kafka, Zookeeper, and Kafka Connect all have link:/docs/monitoring[built-in supp
 |`NumberOfDisconnects`
 |`long`
 |The number of disconnects by the MySQL connector.
+
+|`NumberOfCommittedTransactions`
+|`long`
+|The number of processed transactions that were committed.
+
+|`NumberOfRolledBackTransactions`
+|`long`
+|The number of processed transactions that were rolled back and not streamed.
+
+|`NumberOfNotWellFormedTransactions`
+|`long`
+|The number of transactions that has not conformed to expected protocol `BEGIN` + `COMMIT`/`ROLLBACK`. Should be `0` under normal conditions.
+
+|`NumberOfLargeTransactions`
+|`long`
+|The number of transactions that has not fitted into the look-ahead buffer. Should be significantly smaller than `NumberOfCommittedTransactions` and `NumberOfRolledBackTransactions` for optimal performance.
```

Comment:
s/has/have/

Reply:
Can you clarify that those new options only will be enabled when the buffering is enabled (buffer size > 0)?